### PR TITLE
fix: tg-forward

### PR DESCRIPTION
**Pull request by Decodetalkers**
replace tg-forward block to ''

this make tg-forward can be shown

Log:

### CODE REVIEW

**Comment by Decodetalkers** [comment]
@krille-chan I have fixed tg-forward, make the part visible, can you review it?

**Comment by krille-chan** [comment]
Can you elaborate more what this is about?

**Comment by Decodetalkers** [comment]
telegram has a message is <tg-forward>, the message my side received is like 
```
<tg-forward>
    <blockquote>
message content
   </blockquote>
</tg-forward>
```
because the xml has `<tg-forward>` part, the part of <blockquote> can not be rendered, it become empty, then I cannot view the forward message here

**Comment by q234rty** [comment]
I guess this fixed #507 ?

File: lib/pages/chat/events/html_message.dart [hunk 1]
```diff
@@ -34,15 +34,34 @@ class HtmlMessage extends StatelessWidget {
...
+        .replaceAll(
+          RegExp(
+            '<mx-reply>.*</mx-reply>',
+            caseSensitive: false,
+            multiLine: false,
+            dotAll: true,
+          ),
+          '',
+        )
+        .replaceAll(
```
Comment (krille-chan) [inline lib/pages/chat/events/html_message.dart:47]: I don't think we need two replaceAll methods here. With regex it should be possible to do it in one by selecting the content inside of tg-forward.

**Comment by krille-chan** [comment]
Instead of doing an expensive `.replaceAll()` action on all messages, I would prefer to just add `tg-forward` as another allowed tag and render it the same way as blockquotes

**Pull request closed without merging**